src/content/docs/browser/new-relic-browser/browser-apis/actiontext.mdx

```diff
@@ -13,8 +13,8 @@ redirects:
 ---
 
 <Callout variant="important">
-To use this API, you need the Browser Pro+SPA edition of the browser agent.
-  </Callout>
+To use this API, you need the Browser Pro+SPA edition of the browser agent. If you are using the NPM package, you will need to include the `spa` feature.
```

Comment:
```suggestion
To use this API, you need the Browser Pro+SPA edition of the browser agent. If you're using the [NPM package to install the browser agent](/docs/apis/nerdgraph/examples/combining-npm-nerdgraph/), you must include the `spa` feature.
```
@patrickhousley How do they "include the `spa` feature"? Let's add a bit more detail or linked instructions on how to do this.